src/services/PostValidationService.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+class PostValidationService
+{
+    static public function TitleValidation(string $title): bool
+    {
+        if (strlen($title) === 0 || strlen($title) > 30) {
+            $_SESSION['titleError'] = 'Title cannot exceed 30 characters.';
+            return false;
+        }
+        unset($_SESSION['titleError']);
+        return true;
+    }
+
+    static public function ContentValidation(string $content): bool
+    {
+        if (strlen($content) < 50 || strlen($content) > 1000) {
+            $_SESSION['contentError'] = 'Content must be between 50 and 1000 characters.';
+            return false;
+        }
+        unset($_SESSION['contentError']);
+        return true;
+    }
```

Comment:
Treat this comment as optional.

It would be better if the session stuff were handled outside this class. Partly for simplicity (keep each class as simple as possible), but also for re-usablity. 

What if some situation cropped up in the future where you wanted to validate some data without doing anything in the session? Or with different error messages?